### DISCUSSION TITLE
Photoprism not recognise everything in the .env file

### DISCUSSION THREAD

**Discussion by Tech-dea** [photoprism/photoprism#4419] [Q&A]
So I have all these below in my .env file... however, funny thing is it kind of recognise some of the things if I run `photoprism config` (e.g. it does recognise originals path, storage path, and originals limit, but not index workers, index schedule and default timezone. I had been restarting Photoprism for multiple times... Not sure what was the issue here...?

```
PHOTOPRISM_HTTP_HOST='0.0.0.0'
PHOTOPRISM_HTTP_PORT='2342'
PHOTOPRISM_SITE_CAPTION='Samuel's Photo'
PHOTOPRISM_STORAGE_PATH='/mnt/apps/photoprism/storage'
PHOTOPRISM_ORIGINALS_PATH='/mnt/samuel'
PHOTOPRISM_IMPORT_PATH='/opt/photoprism/photos/import'
PHOTOPRISM_INDEX_WORKERS='4'
PHOTOPRISM_INDEX_SCHEDULE='0 12 * * *'
PHOTOPRISM_DEFAULT_TIMEZONE='Asia/Hong_Kong'
PHOTOPRISM_ORIGINALS_LIMIT='100000'
```

**Comment by pr0fsmith** [comment]
You ever fix this?
